src/main/java/com/example/javawebcourse/featuretoggle/FeatureToggles.java

```diff
@@ -0,0 +1,17 @@
+package com.example.javawebcourse.featuretoggle;
+
+import lombok.Getter;
+
+@Getter
+public enum FeatureToggles {
```

Comment:
```suggestion
public enum FeatureToggle {
```

enums are mostly named in singular, such as Color or FeatureToggle